### DISCUSSION TITLE
User table inconsistent data

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#10311] [Help] [answered]
Hello,

Budibase Cloud , Version : 2.5.2

User information in live app user table is different from the list of the assigned users to app. 

More than 16 users are provided app access , as shown below 

![image](https://user-images.githubusercontent.com/42922833/232089857-ab082ae2-aa6b-48fc-b09c-e4c300de5837.png)

snapshot from live app screen displaying data from users table (no filter applied) , has only subset of users.  Users who are not in the table cannot access app. They were able to access before. 

![image](https://user-images.githubusercontent.com/42922833/232089887-0cbb6b06-f064-454e-9b7a-eba58ea68624.png)

Some of the users have custom role assigned.
App was created from export of existing app and and import.
Users login through organization SSO and NO users were added through API.

Thanks,

**Comment by andz-bb** [comment, marked as answer]
hey @applicationuser, hopefully this has been resolved for you over email. Let me know if you see any more issues like this 👍

**Reply by applicationuser** [reply]
Thank you @andz-bb  , Issue resolved.
